chapters/ru/_toctree.yml

```
@@ -30,3 +30,12 @@
   - local: chapter2/1
     title: Введение
 
+
+- title: 3. Fine-tuning предобученной модели
+  sections:
+  - local: chapter3/1
+    title: Введение
+  - local: chapter3/2
+    title: Предобработка данных
+  – local: chapter3/3
```

We need to fix this character to build the content :)
```suggestion
  - local: chapter3/3
```